src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -96,7 +96,7 @@ function ReportActionItemEmojiReactions(props) {
                 };
 
                 const onReactionListOpen = (event) => {
-                    reactionListRef.current.showReactionList(event, popoverReactionListAnchor.current, reaction.emoji, props.reportActionID);
+                    reactionListRef.current.showReactionList(event, popoverReactionListAnchor.current, reaction.emojiName, props.reportActionID);
```

Comment:
we could use available variable `reactionEmojiName`

Reply:
This is done